index.html

```diff
@@ -11,6 +11,23 @@
         <div class="container">
             <h3>Rolodex</h3>
 
+            <div class="card">
+                    <section class="thumbnail">
+                        <img width="100" src="image/Untitled-1.png" alt="shawn's thumbnail" />
```

Comment:
add my image back and we are good!

Reply:
you updated alt but I don't think my image was named that so be sure to update src.